uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -557,6 +565,8 @@ def test_enable_installs_only_livepatch_snap_when_absent_but_snapd_present(
             mock.call("/usr/bin/snap"),
         ]
         assert expected_calls == m_which.call_args_list
+        assert m_snap_proxy.call_count == 1
```

Comment:
Do we have unittest explicit coverage of the commands passed to subp?
If not can we add assertions of m_snap_proxy.call_args_list  and m_livepatch_proxy

Reply:
No, I will add some unittest for the functions instead